SampleBrowser/SampleBrowser/UI/Browser/BrowserViewController.swift

```diff
@@ -158,6 +162,10 @@ class BrowserViewController: UIViewController,
                                                     canGoForward: canGoForward)
     }
 
+    func didLoad(pageMetadata: WebEngine.EnginePageMetadata) {
+        // TODO: Handle page metadata [?]
```

Comment:
I should have added that delegate call in that PR, sorry! For the question there, I am not sure if we want to have the page metadata fetch in the `SampleBrowser`. We currently don't have tabs over there, and so there's no tab favicon to show anywhere. We could have some fancy stuff to display the favicon, as a test of that implementation, but that seems like a lot of overhead for now maybe
```suggestion
    func didLoad(pageMetadata: EnginePageMetadata) {
        // TODO: Handle page metadata [?]
```

Reply:
> We could have some fancy stuff to display the favicon, as a test of that implementation, but that seems like a lot of overhead for now maybe

Makes sense, for now I'll just plan on leaving this alone and can revisit it later. Unless you have something specific you want to do here now, LMK.